content/about/releases.md

```diff
@@ -0,0 +1,18 @@
+---
+authors: ["kokkos-team"]
+title: "Releases"
+date: "2024-08-11"
+tags: ["Release"]
+---
+
+# Kokkos Ecosystem release page
```

Comment:
Should we say something about the license under which we distribute Kokkos?

Reply:
Yes, just updated with the section below on licensing for Kokkos and Kokkos Kernels.